src/Mod/TechDraw/App/DrawView.cpp

```diff
@@ -439,6 +439,28 @@ DrawView *DrawView::claimParent() const
     return getCollection();
 }
 
+//! return *unique* list of DrawView derived items which consider this DVP to be their 'owner'
+//! if a dimension has two references to this dvp, it will appear twice in the inlist, so we need to
+//! pick out duplicates.
+std::vector<TechDraw::DrawView*> DrawView::getUniqueChildren() const
+{
+    std::vector<TechDraw::DrawView*> result;
```

Comment:
No need to specify the `TechDraw` namespace as we have `using namespace TechDraw` on line 50.

```suggestion
std::vector<DrawView*> DrawView::getUniqueChildren() const
{
    std::vector<DrawView*> result;
```